src/main/kotlin/net/leanix/vsm/sbomBooster/configuration/PropertiesConfiguration.kt

```diff
@@ -19,6 +19,8 @@ data class PropertiesConfiguration(
     val devMode: Boolean,
     val httpProxy: String?,
     val httpsProxy: String?,
+    @field:NotNull
+    val allowNoComponentSboms: Boolean,
```

Comment:
Shall we use:
val allowNoComponentSboms: Boolean = true, instead of NotNull annotation?

Reply:
Fixed in the latest version.